Includes/findXBE.cpp

```diff
@@ -1,31 +1,37 @@
 #include "findXBE.h"
 #include <algorithm>
+#include <cstdio>
+#include <windows.h>
```

Comment:
This `#include` is NXDK specific - will (probably(?)) break compilation under Linux (apart from other issues such as an out-of-date CMakeLists.txt ...)

Reply:
Fixed.